### PR TITLE
Feature/3 header

### LINKED ISSUE #3: Create Header
Props:
- onFilterChange
- onClickNewEvent

### PR DESCRIPTION
**Pull request by al-ros**
Screenshot_9

create header+styles

### CODE REVIEW

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import './Button.css'
+import PropTypes from 'prop-types'
 
-const Button = ({onClick, label}) =>
-    <button className="button" onClick={onClick}>{label}</button>
+const Button = ({onClick, label, className}) =>
+    <button className={`button ${className}`} onClick={onClick}>{label}</button>
```
Comment (roman-vabishchevych) [inline src/components/Button/Button.jsx]: for managing classnames better to use 'classnames' library

https://github.com/JedWatson/classnames

File: src/components/Dropdown/Dropdown.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import PropTypes from 'prop-types';
 import './Dropdown.css';
 
-const Dropdown = ({ options = [], value, onChange }) => {
+const Dropdown = ({ options = [], value, onChange, className }) => {
   return !!options.length && (
-    <select value={ value } onChange={ onChange } className="dropdown">
+    <select value={ value } onChange={ onChange } className={`dropdown ${className}`}>
```
Comment (roman-vabishchevych) [inline src/components/Dropdown/Dropdown.jsx]: the same - https://github.com/JedWatson/classnames

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import './Header.scss'
```
Comment (roman-vabishchevych) [inline src/components/Header/Header.jsx]: use import scss at the end of imports

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+const DROPDOWN_DUMMY_OPTIONS = [
+    { label: 'Item 1', value: 1 },
+    { label: 'Item 2', value: 2 },
+    { label: 'Item 3', value: 3 }
+  ]
+
+const Header = ({onClickNewEvent, filterValue, onFilterChange}) => {
+    return (
+        <header className='header'>
```
Comment (roman-vabishchevych) [inline src/components/Header/Header.jsx]: `'` -> `"` for string attributes

File: src/components/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            <h1 className='header__title'>Calendar</h1>
+            <Dropdown className='header__filter' options={DROPDOWN_DUMMY_OPTIONS} 
+                value={filterValue}
+                onChange={(e) => onFilterChange(e.target.value)} />
+            <Button className='header__button' onClick={onClickNewEvent} label="New event +"/>
+
```
Comment (roman-vabishchevych) [inline src/components/Header/Header.jsx]: remove empty lines

File: src/components/Header/Header.scss [hunk 6]
```diff
@@ -0,0 +1,12 @@
+.header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 0 -.5rem;
+    &__title {
```
Comment (roman-vabishchevych) [inline src/components/Header/Header.scss:7]: add new line before the selector

File: src/components/Header/Header.scss [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+        flex-grow: 2;
+    }
+    &__title, &__filter, &__button {
```
Comment (roman-vabishchevych) [inline src/components/Header/Header.scss:11]: the same - new line

**Review by roman-vabishchevych — CHANGES_REQUESTED** [review]

**Comment by al-ros** [comment]
@roman-vabishchevych all changes are done

**Review by roman-vabishchevych — APPROVED** [review]

**Pull request merged**